Master40.DB/Data/Initializer/MasterDBInitializerBasic.cs

```diff
@@ -204,8 +204,12 @@ public static void DbInitialize(MasterDBContext context)
             var mapping2 = new Mapping { From = "Articles.Name", To = "Material.Name" };
             context.Add(mapping2);
             context.SaveChanges();
-
-
+            var mapping3 = new Mapping { From = "Articles.ToPurchase", To = "Material.InhouseProduction", ConversionFunc = "BoolToLong", ConversionArgs = "true" };
```

Comment:
Could be done inline or as Array
var mappings  = new Mapping[] { new Mapping { .... }, Mapping { . . . . } }
and added to context with 
context.AddRange(mappings);